implementations.md

```diff
@@ -19,3 +19,21 @@ If you know of any associated projects that are not listed here, please file a p
 ## Testing
 
 * [huawei-openlab/oct](https://github.com/huawei-openlab/oct) - Open Container Testing framework for OCI configuration and runtime
+
+## Hooks
+
+For addressing container issues that are outside the scope of this specification, the following hooks may be useful.
+
+### Linux
+
+#### Networking
+
+* [CNI][], the Container Network Interface.
+  The [`exec-plugins.sh`][cni-exec-plugins.sh] script can be used in both [prestart][] (with `add`) and [poststop][] (with `del`) hooks insert and remove a container from networks [configured][cni-conf] in a host-side directory like `/etc/cni/net.d`.
```

Comment:
minor nit: Can we avoid describing the usage of plugins here and instead just mention `CNI` as an option?
I'm thinking if this PR will become a reason for other net plugin vendors to include their specification here.